ubuntudesign/vanilla-framework
# Add link examples

## Done
Add a bunch of examples for links which were missing

## QA
- Run `gulp jekyll`
- Check out the link examples at:
 - http://127.0.0.1:4000/vanilla-framework/examples/patterns/links/links-soft/
 - http://127.0.0.1:4000/vanilla-framework/examples/patterns/links/links-strong/
 - http://127.0.0.1:4000/vanilla-framework/examples/patterns/links/links-inverted/
